sdk/src/main/java/com/meniga/sdk/models/challenges/MenigaChallenge.java

```diff
@@ -468,9 +471,23 @@ public void onFinished(MenigaChallenge result, boolean failed) {
 	 * Gets all challenges, both new ones and accepted ones, also excluded ones if includeExpired is set to true
```

Comment:
Maybe add a comment explaining the effects of the new parameter, which is set to true.

Reply:
You're right that this one should work like before as it's deprecated now.